deploy/script/katello-debug.sh

```diff
@@ -61,6 +61,13 @@ fi
 add_files /etc/gofer
 add_files /var/log/gofer
 
+#foreman-tasks export
+if hash foreman-rake  2>/dev/null; then
+  echo "Exporting tasks, this may take a few minutes."
+  tasks_filename=`foreman-rake foreman_tasks:export_tasks | tail -n 1 | awk '{print $2}'`
```

Comment:
Do you think this should do a check of the return of the command or that it exists? In the current state (of non existent rake task), it leads you to believe its exporting the tasks successfully in the output and then you end up with nothing without an obvious reason that it failed here.